src/devices/Mcp3428/ConversionResult.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Mcp3428
+{
+    public readonly struct ConversionResult
+    {
+        public ConversionResult(byte channel, short rawData, ResolutionEnum resolution)
+        {
+            Channel = channel;
+            RawData = rawData;
+            Resolution = resolution;
+        }
+
+        /// <summary>
+        /// Gets the channel.
+        /// </summary>
+        /// <value>The channel.</value>
+        /// <autogeneratedoc />
+        public byte Channel { get; }
+
+        /// <summary>
+        /// Gets the raw data.
+        /// </summary>
+        /// <value>The raw data.</value>
+        /// <autogeneratedoc />
+        public short RawData { get; }
+
+        /// <summary>
+        /// Gets the resolution.
+        /// </summary>
+        /// <value>The resolution.</value>
+        /// <autogeneratedoc />
+        private ResolutionEnum Resolution { get; }
+
+        /// <summary>
+        /// Voltage value of the least significant bit of the data
+        /// </summary>
+        /// <value>The LSB value.</value>
+        /// <autogeneratedoc />
+        public double LsbValue => Helpers.LSBValue(Resolution);
```

Comment:
this is super confusing.

IMO this should be renamed to i.e. Scale or store reverse unit and call it i.e. Divisor/VoltageDivisor as long (instead of small double store the factor you divide by). Voltage could still be double.

Reply:
I renamed it to Accuracy it's the same value but I hope it's less confusing.
I introduced a VoltageDivisor property as it fits into ushort and removed the Resolution as it's redundant this way.